.github/workflows/update-data.yml

```
@@ -0,0 +1,39 @@
+name: 'Update static data'
+on:
+  schedule:
+    - cron: '0 0 * * 0' # Every Sunday at midnight
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: 14
```

```suggestion
  NODE_VERSION: 16
```

Maybe update to node 16?

---

Sure! I only chose 14 as it's what all other workflows use.